src/main/java/org/eln2/mc/common/blocks/CellTileEntity.kt

```diff
@@ -140,11 +140,12 @@ class CellTileEntity(var pos : BlockPos, var state: BlockState): BlockEntity(Blo
             }
         }
 
-        logger.info("Remove nanoseconds: $nanoTime")
+        Eln2.LOGGER.info("Remove nanoseconds: $nanoTime")
 
         cell.graph.destroyAndRemove()
 
-        val t = thread (start = true) {
+        thread (start = true) {
+            // TODO: We're creating a thread... and then HttpPlot is creating it's own thread for every item in the list. This should be cleaned up, it won't scale once we release.
```

Comment:
We're creating a thread... and then HttpPlot is creating it's own thread for every item in the list. This should be cleaned up, it won't scale once we release.

I did make it so that it wouldn't make child threads if the HttpPlot client wasn't connected which saves creating useless threads. This should help performance if you are not using the debug harness.